From efdc0e9f7f25db82e8cbf8b579cc978b7a2da514 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kuklin=20Istv=C3=A1n?= <kuklinistvan@zoho.com>
Date: Thu, 12 Mar 2020 18:11:16 +0100
Subject: [PATCH] Orcania Yder from scratch bug

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e813e5c..c356cb4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -251,9 +251,9 @@ if (SEARCH_ORCANIA_U)
         download_project(PROJ orcania # ... otherwise, download archive
             URL "https://github.com/babelouest/orcania/archive/v${ORCANIA_VERSION_REQUIRED}.tar.gz"
             QUIET)
-        add_subdirectory(${orcania_SOURCE_DIR} ${orcania_BINARY_DIR})
+        #add_subdirectory(${orcania_SOURCE_DIR} ${orcania_BINARY_DIR})
         include_directories(${orcania_SOURCE_DIR}/include)
-        add_dependencies(ulfius orcania)
+        #add_dependencies(ulfius orcania)
         set(ORCANIA_LIBRARIES orcania)
         set(LIBS ${LIBS} ${ORCANIA_LIBRARIES})
         include_directories(${orcania_BINARY_DIR})
-- 
2.25.1

