--- CMakeLists.txt	2020-09-18 14:10:00.569991788 +0200
+++ CMakeLists.txt.mod	2020-09-18 14:09:48.959991813 +0200
@@ -19,9 +19,6 @@
 project(ulfius C)
 
 set(CMAKE_C_STANDARD 99)
-if (NOT MSVC)
-    set(CMAKE_C_FLAGS "-Wall -Werror ${CMAKE_C_FLAGS}")
-endif()
 
 # library info
 
@@ -61,7 +58,7 @@
 
 # cmake modules
 
-set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake-modules)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake-modules)
 
 include(GNUInstallDirs)
 include(CheckSymbolExists)
@@ -109,8 +106,7 @@
 option(WITH_GNUTLS "GNU TLS support" ON)
 
 if (WITH_GNUTLS)
-    include(FindGnuTLS)
-    find_package(GnuTLS REQUIRED)
+    find_package(gnutls REQUIRED)
     if (GNUTLS_FOUND)
         set(LIBS ${LIBS} ${GNUTLS_LIBRARIES})
     endif ()
@@ -141,9 +137,8 @@
     set(U_DISABLE_GNUTLS ON)
 endif ()
 
-include(FindMHD)
-find_package(MHD ${MHD_MIN_VERSION} REQUIRED)
-if (MHD_FOUND)
+find_package(libmicrohttpd ${MHD_MIN_VERSION} REQUIRED)
+if (libmicrohttpd_FOUND)
     set(LIBS ${LIBS} ${MHD_LIBRARIES})
     include_directories(${MHD_INCLUDE_DIRS})
     if (MHD_VERSION_STRING VERSION_LESS "0.9.53")
@@ -160,8 +155,7 @@
 option(WITH_CURL "Use Curl library" ON)
 
 if (WITH_CURL)
-    include(FindCURL)
-    find_package(CURL REQUIRED)
+    find_package(curl REQUIRED)
     if (CURL_FOUND)
         set(LIBS ${LIBS} ${CURL_LIBRARIES})
         include_directories(${CURL_INCLUDE_DIRS})
@@ -174,12 +168,11 @@
 option(WITH_JANSSON "Use jansson library" ON)
 
 if (WITH_JANSSON)
-    include(FindJansson)
     set(JANSSON_MIN_VERSION 2.4)
-    find_package(Jansson ${JANSSON_MIN_VERSION} REQUIRED)
-    if (JANSSON_FOUND)
-        include_directories(${JANSSON_INCLUDE_DIRS})
-        set(LIBS ${LIBS} ${JANSSON_LIBRARIES})
+    find_package(jansson ${JANSSON_MIN_VERSION} REQUIRED)
+    if (jansson_FOUND)
+        include_directories(${jansson_INCLUDE_DIRS})
+        set(LIBS ${LIBS} ${jansson_LIBRARIES})
         set(U_DISABLE_JANSSON OFF)
     endif ()
 else ()
